### DISCUSSION TITLE
778

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#950] [Comments]
# 778

연휴에 외부의 주니어 개발자분을 만나서 커피챗을 진행했다. 커피챗에서 받은 질문은 "향로님은 4개의 회사를 다니셨고 3번의 이직을 하셨는데, 3번다 이직이 만족스러웠냐?" 였다. 나는 3번의 이직이 모두 만족스러웠다. 그 타이밍에 그 회사들을 간 것에 대해 정말 잘한 선택이라고 생각하고 있고 실제로 이후의 커리어에도 계속 도움이 되었다. 어떻게 그럴수 있었을까? 에 대해 답변 드린 내용이다. 보통 "후회없는 이직을 하려면 어떻게 해야하느냐", "언제 이직하면 좋으냐" 라는 질문을 자주 받는데, 여기엔 여러 답변들이 있을 수 있다. 더이상 회사에서 성장하지 못한다고 느낄때 회사가 더이상 성장하지 않는다고 느낄때 회사의 조직 문화가 너무 안맞다고 느낄때 사람간의 관계가 너무 힘들때 등등 퇴사/이직의 트리거는..

https://jojoldu.tistory.com/778

**Comment by krapie** [comment]
여러모로 현타를 느끼던 와중에 타이밍 좋게 이 글을 보게 되었는데요, 스스로 많이 고민해보게 되네요. 좋은 글 써주셔서 말 감사합니다!

**Comment by fe-dudu** [comment]
항상 글만 읽어오다가 처음 댓글을 남깁니다.
향로님 글을 읽고 제 자신을 많이 돌아보기도 하고 동기부여도 얻고 갑니다.  :-)
좋은 글 써주셔서 감사합니다 !

**Comment by byeongsoon** [comment]
"그러니 지금 회사가 다니기 싫다면, 바로 다음 회사를 찾는 것이 아니라 지금 회사를 다닐만한 회사로 개선해보는 것을 먼저 고려해보자."

이 문장을 읽고서 생각이 많아지는 것 같습니다. 좋은 글 감사합니다.
